renn/utils.py

```diff
@@ -137,3 +139,85 @@ def one_hot(labels, num_classes, dtype=jnp.float32):
     one_hot_labels: array with shape (num_examples, num_classes).
   """
   return jnp.array(jnp.array(labels)[:, None] == jnp.arange(num_classes), dtype)
+
+
+def select_output(sequences, indices):
```

Comment:
Maybe just call this `select`, since it can be applied to anything (not just the output)?

We should also add some simple tests for this function ... I find that this kind of array manipulation / filtering code benefits the most from simple unit tests.

Reply:
Agree, I called it select and renamed the variable `selected_outputs` inside the function to `selected_elements`.  

For the test, I'll open an issue.